bosh-stemcell/spec/stemcells/gpu_spec.rb

```diff
@@ -0,0 +1,238 @@
+require 'spec_helper'
+
+describe 'Ubuntu 22.04 GPU stemcell image', stemcell_image: true do
+
+    describe 'record use of privileged programs (CIS-8.1.12)'  do
+      let(:privileged_binaries) {
+        command("find /bin /sbin /usr/bin /usr/sbin /boot -xdev \\( -perm -4000 -o -perm -2000 \\) -type f")
+          .stdout
+          .split
+      }
+
+      describe file('/etc/audit/rules.d/audit.rules') do
+        its(:content) do
+          privileged_binaries.each do |privileged_binary|
+            should match /^-a always,exit -F path=#{privileged_binary} -F perm=x -F auid>=500 -F auid!=4294967295 -k privileged$/
+          end
+        end
+      end
+    end
+
+    context 'System command files must have mode 0755 or less permissive (stig: V-38469)' do
+        describe command('find -L /bin /usr/bin /usr/local/bin /sbin /usr/sbin /usr/local/sbin -perm /022 -type f') do
+          its (:stdout) { should eq('') }
+        end
+    end
+
```

Comment:
These tests appear to be word-for-word identical to tests in `stemcell_shared_examples.rb` and `os_image_shared_examples.rb`. Is there a reason why they've been duplicated here?